games-action/minetest/minetest-0.4.12.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit cmake-utils eutils gnome2-utils user vcs-snapshot
```

Comment:
It's about removing the inheritance of games.eclass which was designed by the games project to have _consistent behavior_ of games across the tree. Now, some devs think it is a good idea to just break this consistency instead of _properly_ deprecating the eclass. I think that is bad. And I have very strong opinion about it, although I would vote for deprecating it. But if we do that, we have to do that consistently, not randomly.